vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,150 @@
+10. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources. 
+11. Go to `EntandoApp → my-app` to check the status of the deploy
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 2 - PostgreSQL
+ For this scenario we'll prepare a more production-like configuration. Here we'll switch from an embedded on-disk database to a dedicated PostgreSQL database. For this scenario we'll start where step 6 ends in the previous scenario. If you already ran Scenario 1 you can either go to that project and remove the EntandoApp and ProvidedCapabilities via `Installed Operators → Entando Operator` or prepare a new project using steps 1-5 above.
+ 
+Now let's create a new application, this time using PostgreSQL.
+1. Go to `EntandoApp` and click `Create instance`
+   - Keep the default `my-app` for your application name or select your own
+   - Select the EntandoApp version: `7.0`
+2. Set the `Ingress Host Name` as in Scenario 1 above
+3. Keep the default `DBMS` as `postgresql`  
+4. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources just as in Scenario 1 but with the addition of a PostgreSQL database deployment.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 3 - PostgreSQL plus OpenShift SSL
+For this scenario we'll setup PostgreSQL, just like Scenario 2, but also enable SSL using OpenShift's internal Certificate Authority (CA). As a starting point, you can either remove the EntandoApp and ProvidedCapabilities using the Operator or you can prepare a new project per steps 1-5 in Scenario 1.
+
+1. Using your browser, examine and save the SSL certificate for your environment. For example, in Chrome, go to any OpenShift console page, click on `View Site Information` next to the URL (it may be labeled `Not secure` ) then go to `Details -> Copy to File` and save the certificate as a Base-64 encoded X.509 file, e.g. `openshift.cer`.
+2. Go to `Workloads → Secrets → Create` and select `Key/value secret`
+3. Set the `Secret Name`, e.g. `entando-ca-cert-secret` 
+4. Set the `Key`, e.g. `openshift.cer`
+5. Set the `Value` by clicking `Browse...` and loading the file you saved in Step 1
+6. Click `Create`
+7. Next go to `Workloads → ConfigMaps` and create or update the ConfigMap named `entando-operator-config`. This is the ConfigMap used by the Operator to configure the deployments. You can [download the Entando Operator template](../devops/entando-operator.md#add-a-new-configmap) as a starting point. Set the "data/entando.ca.secret.name" to match the name from step 3.
+```yaml
+data:
+ entando.ca.secret.name: entando-ca-cert-secret
+ entando.use.auto.cert.generation: 'true'
+```
+8. Click `Create`
+
+Now let's create a new application, just like in Scenario 2 but with OpenShift SSL in place.  
```

Comment:
application, just like in Scenario 2 but -> application similar to Scenario 2, but